node/dataSources/fixedPrice.ts

```diff
@@ -0,0 +1,50 @@
+import {
+  RequestOptions,
+  RESTDataSource
+} from 'apollo-datasource-rest'
+import { forEachObjIndexed } from 'ramda'
+
+export class FixedPriceDataSource extends RESTDataSource<Context> {
+  constructor() {
+    super()
+  }
+
+  public fixedPrices = (itemId: string = '') => {
+    return this.get(`/${itemId}/fixed`)
```

Comment:
This API is private, the store can't use this resolver if the user isn't logged.

Reply:
What is the purpose of making this private since user is allowed to buy multiple quantities of items without login in to the system?
or 
How do you suggest to make this private?